### ISSUE TITLE
Add madness library

### ISSUE THREAD

**Issue by ncook882** [radiasoft/download#609]
For the Structwake project, we'd like to have the [madness](https://github.com/m-a-d-n-e-s-s/madness/) library made available on Jupyter.

**Comment by ncook882** [comment]
Sometime next week would be great.

I recommend a fairly basic install of the release version. Including some performance libraries could be nice (e.g. Intel TBB, MKL, and Google Performance Tools). I don't think we need any external physics libraries or other external features. Let me know if you want more input.

**Comment by e-carlin** [comment]
Thanks. We'll need a week to get it through our release cycle. So, alpha + beta next week and prod the following. I'll start working on it this afternoon.

**Comment by e-carlin** [comment]
TBB doesn't work with mpi > 1 process ([from their docs](https://github.com/m-a-d-n-e-s-s/madness/blob/master/INSTALL.md#warning-about-fast-memory-allocators)). I want to confirm that you want it enabled?

**Comment by ncook882** [comment]
I don't think we plan to experiment with fast memory allocators, so we should be able to use standard TBB task management with MPI > 1. So I suggest we keep it enabled. If there's an issue in testing that suggests it won't work in parallel, we can revisit.

**Comment by e-carlin** [comment]
👍 
The version of tbb in the fedora repos is too old to work with the latest version of madness. And intel mkl will require enabling intel repos to install (or figuring out how to build from source). So, in the interest of expediency I'm going to cut a release with just gperftools and then when I have more time loop back to tbb and mkl. Is that ok?

**Comment by ncook882** [comment]
> 👍 The version of tbb in the fedora repos is too old to work with the latest version of madness. And intel mkl will require enabling intel repos to install (or figuring out how to build from source). So, in the interest of expediency I'm going to cut a release with just gperftools and then when I have more time loop back to tbb and mkl. Is that ok?

All good. Thanks!

**Comment by ncook882** [comment]
@robnagler looks like madness still isn't installed on Jupyter. @dtabell tested on a fresh instance. Should we expect it to be available with a new release?

**Comment by robnagler** [comment]
For some reason there wasn't a prod release last week. It's on deck to be pushed to prod on Monday. I'll make sure that happens.

**Comment by robnagler** [comment]
This is on production:

jupyter$ znemo 
about to call bind
MADNESS runtime initialized with 39 threads in the pool and affinity OFF

================================================================================
    ZNEMO -- complex Hartree-Fock using numerical exponential molecular orbitals
================================================================================ 

starting at time 0.0
A madness exception occurred. Place a break point at madness::exception_break to debug.
terminate called after throwing an instance of 'madness::MadnessException'
  what():  MADNESS CHECK FAILED: 
Aborted

**Comment by dtabell** [comment]
Thanks, Rob,
Can you point me to the include dir?
Dan

heat> ls
heat.cc
heat> gcc -o heat heat.cc 
heat.cc:59:10: fatal error: madness/mra/mra.h: No such file or directory
   59 | #include <madness/mra/mra.h>
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.

**Comment by robnagler** [comment]
`~/.local/include`
